docs/self-managed/zeebe-deployment/operations/cluster-scaling.md

```diff
@@ -315,10 +1209,170 @@ The response would show that partitions are distributed to new brokers:
+      "port": 26501,
+      "partitions": [
+        {
+          "partitionId": 1,
+          "role": "leader",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 2,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 6,
+          "role": "follower",
+          "health": "healthy"
+        }
+      ],
+      "version": "8.8.0-SNAPSHOT"
+    },
+    {
+      "nodeId": 0,
+      "host": "zeebe-scaling-demo-zeebe-0.zeebe-scaling-demo-core",
+      "port": 26501,
+      "partitions": [
+        {
+          "partitionId": 1,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 5,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 6,
+          "role": "follower",
+          "health": "healthy"
+        }
+      ],
+      "version": "8.8.0-SNAPSHOT"
+    },
+    {
+      "nodeId": 3,
+      "host": "zeebe-scaling-demo-zeebe-3.zeebe-scaling-demo-core",
+      "port": 26501,
+      "partitions": [
+        {
+          "partitionId": 2,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 3,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 4,
+          "role": "leader",
+          "health": "healthy"
+        }
+      ],
+      "version": "8.8.0-SNAPSHOT"
+    },
+    {
+      "nodeId": 5,
+      "host": "zeebe-scaling-demo-zeebe-5.zeebe-scaling-demo-core",
+      "port": 26501,
+      "partitions": [
+        {
+          "partitionId": 4,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 5,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 6,
+          "role": "leader",
+          "health": "healthy"
+        }
+      ],
+      "version": "8.8.0-SNAPSHOT"
+    },
+    {
+      "nodeId": 4,
+      "host": "zeebe-scaling-demo-zeebe-4.zeebe-scaling-demo-core",
+      "port": 26501,
+      "partitions": [
+        {
+          "partitionId": 3,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 4,
+          "role": "follower",
+          "health": "healthy"
+        },
+        {
+          "partitionId": 5,
+          "role": "leader",
+          "health": "healthy"
+        }
+      ],
+      "version": "8.8.0-SNAPSHOT"
+    }
+  ],
+  "clusterSize": 6,
+  "partitionsCount": 6,
+  "replicationFactor": 3,
+  "gatewayVersion": "8.8.0-SNAPSHOT",
+  "lastCompletedChangeId": "2"
+}
+```
+
+</details>
+
 ## Scale down
 
 We will explain how to scale down a Zeebe cluster via an example of scaling from cluster size 6 to cluster size 3. We assume the cluster is running with 6 brokers.
 
+:::note
```

Comment:
❓️should be use something more important? This is a pretty big limitation.

Reply:
Let's use a `:::warning`